### PR TITLE
Don't use LESS compiler in production at all

### PR DESCRIPTION
**Pull request by humphd**
This stops us from running pure CSS in a `dist/` build through LESS, which is expensive in the startup path.

### CODE REVIEW

File: src/view/ThemeManager.js [hunk 1]
```diff
@@ -233,20 +234,26 @@ define(function (require, exports, module) {
...
+
         if (theme) {
-            require(['text!' + theme.file._path], function(lessContent) {
-                lessifyTheme(lessContent.replace(commentRegex, ""), theme)
-                .then(function (content) {
-                    var result = extractScrollbars(content);
-                    theme.scrollbar = result.scrollbar;
-                    return result.content;
-                })
-                .then(function (cssContent) {
```
Comment (gideonthomas) [inline src/view/ThemeManager.js]: this is such a misuse of promises.

**Comment by humphd** [comment]
I realized this this means we don't actually need to load `less.min.js` at all, which saves 144K!  @flukeout can you test this?  Also, @gideonthomas can you take another look?  I think this is safe, but want to be sure.

Also, did we change the dark theme, or is my patch doing something weird in `dist/` builds?

When you test this, test both `src/` and `dist/` builds, and with `dist/` make sure you wipe your cache/service worker:

screen shot 2017-04-21 at 12 09 55 pm

**Comment by gideonthomas** [comment]
@humphd yeah I'm seeing weird content highlighting in the dist build.

I also see the following in terms of network usage for dist builds:
1. With your patch - 1.7MB transferred, 195 requests
2. Without your patch - 2.0MB transferred, 195 requests

It's weird that the number of requests are the same.

**Comment by humphd** [comment]
The number of files will remain the same for `dist/` builds, since `less.min.js` was being added to `thirdparty.min.js`, which we still download.

What does "weird content highlighting" mean?  Can you show some screenshots?

**Comment by gideonthomas** [comment]
screen shot 2017-04-24 at 10 30 28 pm

**Comment by humphd** [comment]
I see what the issue is here: I'm not loading the light and dark theme files correctly.  I'll fix it when I get a minute.  This will be a good change for reducing our startup size/complexity.  Thanks for catching the issue.

**Comment by humphd** [comment]
@gideonthomas I fixed the issue.  The problem was that when the theme got rendered by LESS, it rewrote every selector to include `#editor-holder` as a parent.  Instead of doing that dynamically at runtime, I just did it in the theme files, and then it works in both cases.

This should be good now, can you test and see if you're OK with it?

**Comment by gideonthomas** [comment]
Nice! This works now!

**Pull request merged**